provisioning/provisioning-service-client/src/main/java/com/microsoft/azure/sdk/iot/provisioning/service/configs/BulkEnrollmentOperationError.java

```diff
@@ -37,53 +38,23 @@
     private static final String REGISTRATION_ID_TAG = "registrationId";
     @Expose
     @SerializedName(REGISTRATION_ID_TAG)
+    @Getter
```

Comment:
```suggestion
    /**
     * Getter for the error registrationId.
     *
     * @return The {@code String} with the registrationId content. It cannot be {@code null}.
     */
    @Getter
```

Just to note that with the @Getter methods you can also include full doc comments that will get generated when the lombok plugin executes for Maven. See

https://projectlombok.org/features/GetterSetter